backend/OpenAPI.yaml

```diff
@@ -204,6 +270,21 @@ components:
         description:
           type: string
           example: "Lightly used gaming laptop, great condition!"
-        message:
+        seller_id:
+          type: string
+          example: "101"
+        pictures:
+          type: array
+          items:
+            type: string
+            example: "https://example.com/image1.jpg"
+        category:
+          type: string
+          example: "Electronics"
+        date_posted:
+          type: string
+          format: date-time
+          example: "2025-02-22T10:00:00Z"
```

Comment:
You can add condition here too. No need for it to be required here like you did for the rest since it is a response so that we can choose to omit fields as we see fit.